Makefile

```diff
@@ -12,36 +7,37 @@ MANPREFIX = $(PREFIX)/share/man
 DOCPREFIX = $(PREFIX)/share/doc/nsxiv
```

Comment:
`=` -> `?=` for MANPREFIX and DOCPREFIX

Reply:
I think it's natural that man and doc goes under the same PREFIX. And one can use `make MANPREFIX=..` to change if needed.

I'm fine keeping this as is, but okay changing it too.

Reply:
Actually, unless examples are directly linked to some documentation documents, it would be nice to be able to install examples separately from documents. On FreeBSD for example we install documentation at $(PREFIX)/share/doc/foobar and examples at $(PREFIX)/share/examples/foobar. From what I know, other systems have similar standards.

Reply:
@0mp Our "document" here is basically just the examples (key-hander and image-info). So you can do `make DOCPREFIX="$PREFIX/share/examples/nsxiv"` to install them under the desired directory.

Reply:
That would create `$PREFIX/share/examples/nsxiv/examples/{key-handler,image-info}` , is that ok or did you want `$PREFIX/share/examples/nsxiv/{key-handler,image-info}` ?

Reply:
@0mp Since we aren't really installing any "docs" but only examples, just changed it to `EGPREFIX` instead.